src/main/webapp/scripts/favouriteOrder.js

```diff
@@ -111,12 +111,12 @@ HBT.Salat.FavouriteOrders = HBT.Salat.FavouriteOrders || function() {
 				var selected = elem.options[elem.selectedIndex].value;
 				if(img.getAttribute("data-isFav") == "true") {
 					if(!setOrder(selected, false)) return;
-					img.src = "/tb/images/Button/whiteStar.svg"
+					img.src = "../images/Button/whiteStar.svg"
```

Comment:
Warum diese Änderung?

Reply:
Weil das Javascript von genau einer JSP-Datei geladen wird, welches sich nun in einem Unterordner befindet. Pfade sind nun relativ zum Pfad der JSP-Seite.